src/System.Runtime.Numerics/src/System/Numerics/BigInteger.cs

```diff
@@ -275,59 +275,107 @@ int IComparable.CompareTo(Object obj)
         // return an array of one byte whose element is 0x00.
         public byte[] ToByteArray()
         {
-            if (_bits == null && _sign == 0)
+            int sign = _sign;
+            if (sign == 0)
```

Comment:
I prefer to wrap even single-statement ifs in braces. But I am not sure if it's our general convention.